java/code/src/com/suse/manager/webui/services/pillar/MinionGeneralPillarGenerator.java

```diff
@@ -135,7 +135,7 @@ else if (chan.isTypeDeb()) {
         if (ConfigDefaults.get().isMetadataSigningEnabled()) {
             chanProps.put("gpgcheck", chan.isGPGCheck() ? "1" : "0");
             // three state field. yes, no or default
-            chanProps.put("repo_gpgcheck", "default");
+            chanProps.put("repo_gpgcheck", "0");
```

Comment:
No, this will turn off the fast repo signature check. At least for "SUSE" systems it would reduce the security.

I would suggest you change not the java code, but the repo template https://github.com/uyuni-project/uyuni/blob/master/susemanager-utils/susemanager-sls/salt/channels/channels.repo

This is where the pillar data generate the .repo file. If you move the 

```
  repo_gpgcheck={{ args['repo_gpgcheck'] }}
  pkg_gpgcheck={{ args['pkg_gpgcheck'] }}
```
section in the "else" tree of `{%- if grains['os_family'] == 'RedHat' %}` it should solve the problem as well.
Maybe you are the expert. Which of these variables are known to "yum" and which to "dnf" ?

Reply:
Btw: this is easy for you to test: look on your Uyuni Server at `/usr/share/susemanager/salt/channels/channels.repo`. When you modify that file, and apply the channels you can directly test your changes.

Reply:
Yes, we talking about this here: https://github.com/uyuni-project/uyuni/issues/2156#issuecomment-630089453

It need to change this:  `repo_gpgcheck=0` and working well.

I will change my commit.

Reply:
new commit, i close this. https://github.com/uyuni-project/uyuni/pull/2236